source/guides/community.rst

```diff
@@ -0,0 +1,81 @@
+=======================================
+Community Engagement Guidelines 
+=======================================
+
+Below is a list of guidelines for maintainers, key contributors and Mattermost, Inc. staff to use when engaging with the broader community in forums, GitHub issues, Twitter, our Mattermost servers and other places of online conversation. 
+
+Best Practices 
+============================================================================
+
+1. Don't offer advice on database modification without a clear warning
+----------------------------------------------------------------------------
+
+Never give DB advice without a warning similar to: `Manual database modification bypass checks in the code and could make a system unrecoverable. It's not recommended and if you do make a change, make sure to back up your DB, and verify changes carefully.`
+
+2. Answer with docs
+----------------------------------------------------------------------------
+
+When answering a common question, always include a link to the docs so the person asking--along with people in future who read your response. If no doc exists, do one of two things: 
+
+1. Add the  the documentation, then answer the question with a link to the information, or 
+2. Answer the question and open a ticket for the missing information to be added to docs.
+
+During the triage process, if the information missed should have obviously been included, assign the ticket to the earliest approver for the doc PR, otherwise assign to doc PR author. 
```

Comment:
This is a little unclear, won't most docs we link to be pretty old, and may potentially have had multiple PRs made to them? Who counts as the doc PR approver? Seems complex for triage